wls-gui-wahllokalsystem/src/types/wls-types/BroadcastMessage.ts

```diff
@@ -0,0 +1,15 @@
+export interface BroadcastMessageToRead {
```

Comment:
Sollte man das nicht besser in eine eigene Datei hinterlegen?

Reply:
ich hatte es vorher so, habe es dann aber beabsichtigt in die gleiche datei geschrieben, mit dem hintergrund, dass mit den anderen services noch sehr viele files mehr dazu kommen und es so ggf etwas übersichtlicher und weniger überladen ist.
ist es besser alles in ein eigenes file zu schreiben und dann ggf weitere unterordner für die jeweiligen services zu erstellen?

Reply:
siehe [4bc3e96](https://github.com/it-at-m/Wahllokalsystem/pull/633/commits/4bc3e96fbc7b49b7c2e798dbcbae3e2fece88409) und [09357c8](https://github.com/it-at-m/Wahllokalsystem/pull/633/commits/09357c8ba997dcdce912fe4eee53c439056a0ea4)